package.json

```diff
@@ -55,13 +55,13 @@
     "karma-sinon-chai": "1.0.0",
     "karma-sourcemap-loader": "0.3.5",
     "karma-webpack": "1.7.0",
+    "lodash.isequal": "^4.5.0",
     "lodash.isfinite": "^3.3.1",
     "lodash.isnumber": "^3.0.3",
-    "lodash.isequal": "^4.5.0",
     "loglevel": "^1.4.0",
     "material-ui": "^0.17",
     "mocha": "2.2.5",
-    "moment": "^2.13.0",
+    "moment": "^2.18.1",
```

Comment:
@adrianq Yes I tried the following
```ts
import moment from 'moment'
import { moment } from 'moment'
import * as moment from 'moment'
```
but there was an issue with the import, I investigated a little bit but I didn't want to spend much time on this. I knew an old workaround, but I then decided to use this wrapper to avoid the hassle